packages/kbn-esql-validation-autocomplete/src/code_actions/actions.test.ts

```diff
@@ -197,68 +258,147 @@ describe('quick fixes logic', () => {
+        );
+      }
+      // it should preserve the arguments
+      testQuickFixes(`FROM index | EVAL rAund(numberField)`, ['round(numberField)'], {
+        equalityCheck: 'include',
+        ...options,
+      });
+      testQuickFixes(`FROM index | STATS AVVG(numberField)`, ['avg(numberField)'], {
+        equalityCheck: 'include',
+        ...options,
+      });
     }
-    // it should preserve the arguments
-    testQuickFixes(`FROM index | EVAL rAund(numberField)`, ['round(numberField)'], {
-      equalityCheck: 'include',
-    });
-    testQuickFixes(`FROM index | STATS AVVG(numberField)`, ['avg(numberField)'], {
-      equalityCheck: 'include',
-    });
   });
 
   describe('fixing wrong quotes', () => {
-    testQuickFixes(`FROM index | WHERE stringField like 'asda'`, ['"asda"']);
-    testQuickFixes(`FROM index | WHERE stringField not like 'asda'`, ['"asda"']);
+    for (const options of [
+      undefined,
+      { relaxOnMissingCallbacks: false },
+      { relaxOnMissingCallbacks: false },
+    ]) {
+      testQuickFixes(`FROM index | WHERE stringField like 'asda'`, ['"asda"'], options);
+      testQuickFixes(`FROM index | WHERE stringField not like 'asda'`, ['"asda"'], options);
+    }
   });
 
   describe('fixing unquoted field names', () => {
-    testQuickFixes('FROM index | DROP any#Char$Field', ['`any#Char$Field`']);
-    testQuickFixes('FROM index | DROP numberField, any#Char$Field', ['`any#Char$Field`']);
+    for (const options of [
+      undefined,
+      { relaxOnMissingCallbacks: false },
+      { relaxOnMissingCallbacks: false },
+    ]) {
+      testQuickFixes('FROM index | DROP any#Char$Field', ['`any#Char$Field`'], options);
+      testQuickFixes(
+        'FROM index | DROP numberField, any#Char$Field',
+        ['`any#Char$Field`'],
+        options
+      );
+    }
+    describe('with no callbacks', () => {
+      describe('with no relaxed option', () => {
+        it('return no result without callbacks and relaxed option', async () => {
+          const statement = `FROM index | DROP any#Char$Field`;
+          const { errors } = await validateQuery(statement, getAstAndSyntaxErrors);
+          const edits = await getActions(statement, errors, getAstAndSyntaxErrors);
+          expect(edits.length).toBe(0);
+        });
+
+        it('return no result without specific callback and relaxed option', async () => {
+          const callbackMocks = getCallbackMocks();
+          const statement = `FROM index | DROP any#Char$Field`;
+          const { errors } = await validateQuery(statement, getAstAndSyntaxErrors, undefined, {
+            ...callbackMocks,
+            getFieldsFor: undefined,
+          });
+          const edits = await getActions(statement, errors, getAstAndSyntaxErrors, undefined, {
+            ...callbackMocks,
+            getFieldsFor: undefined,
+          });
+          expect(edits.length).toBe(0);
+        });
+      });
+      describe('with relaxed option', () => {
```

Comment:
Sorry, I'm not following.
This suite is testing exactly that. I had to write it the long way as callbacks are handled automatically with the `testQuickFixes` utility otherwise.